### ISSUE TITLE
Use device classes for sensors with a duration

### ISSUE THREAD

**Issue by dshokouhi** [simbaja/ha_gehome#312]
Using sensor device class `duration` and a unit of measurement on the time left sensor on a washer will show the UI graph and let the user change the unit if they desire. Really though the graph could be useful here :)

![image](https://github.com/user-attachments/assets/f0501517-df53-4a90-807e-225b1bade629)

**Comment by jawilson** [comment]
@simbaja not sure why this was closed. The commit that includes the merge commit from my PR (85b986a6bd75a7913e7b7d33cb342ef38619f1fa) was orphaned when the `dev` branch was deleted.

**Comment by simbaja** [comment]
My apologies, it got lost when I switched from having a dev branch to just doing things on short-lived branches.  I thought that everything had been merged in, but I guess I missed one.

**Comment by simbaja** [comment]
Can you check the latest dev version (2025.12.0-dev0) and let me know if it all looks good?

**Comment by jawilson** [comment]
> Can you check the latest dev version (2025.12.0-dev0) and let me know if it all looks good?

I switched to the latest release and it seems to working fine. Thank you!

**Comment by kadeschs** [comment]
Whoa! Is this why the timers in all of my equipment dashboards now read thousands of seconds instead of what the devices actually show in minutes everywhere else? Why would we ever want to do this? My washer is going to be done in 2,534 seconds. Ha! What am I missing here? I have to do math on everything or is there a units setting somewhere?

**Comment by simbaja** [comment]
Yep, if you go into the settings of the sensor, you can change the units to
minutes, which will look like the old display.  Enabling these as durations
allows for better support of automations because you can do easy math with
them instead of string parsing.

**Comment by kadeschs** [comment]
Setting the precision to display precision to 18.7 yields shows 18m 42s in config but sensor.ma111279g_laundry_time_remaining state shows 18.7166666666667.  This thing is a huge mess now.

**Comment by rosstroha** [comment]
@simbaja can this be repopened? I came here looking for an existing issue for this. The sensor entities for timers should be a `timer` instead of `sensor`
